api/s2n.h

```diff
@@ -56,6 +56,8 @@ extern int s2n_config_free(struct s2n_config *config);
 extern int s2n_config_free_dhparams(struct s2n_config *config);
 extern int s2n_config_free_cert_chain_and_key(struct s2n_config *config);
 
+extern int s2n_config_init_session_ticket_keys(struct s2n_config *config);
+extern int s2n_config_free_session_ticket_keys(struct s2n_config *config);
```

Comment:
I think it's missing from docs?

Reply:
Oh I meant to add this to s2n_config.h and not in api/s2n.h. Will update.